### PR TITLE
Bump develop branch

### PR DESCRIPTION
**Pull request by paulinebessoles**
* Update gemspec

* Git ignore '.byebug_history'

* Fix tests

* Refactor raw_info specs

* Enhance tests for 'given_name' key

* Add tests for 'email' key

* Update rubocop rules

* Fix rubocop offenses

* Remove unnecessary '.byebug_history' file

* Raise error when site param is undefined or empty

* Check errors

* Add Github Actions workflows

* Bump bundler version

* Bump bundler version

* Fix rubocop offense

* Move 'byebug' requirement

### CODE REVIEW

File: lib/omniauth/publik/version.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 module Omniauth
   module Publik
-    VERSION = "0.1.0"
+    VERSION = "0.0.9"
```
Comment (Quentinchampenois) [inline lib/omniauth/publik/version.rb]: We should bump the current version to `0.1.1`

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
I can update this file if you want

File: lib/omniauth/publik/version.rb (hunk 1 above)
Comment (paulinebessoles) [reply lib/omniauth/publik/version.rb]: I totally agree!

Comment (paulinebessoles) [reply lib/omniauth/publik/version.rb]: I do it right now 😁

**Pull request merged**